rebasehelper/argument_parser.py

```diff
@@ -95,7 +97,8 @@ class CustomArgumentParser(argparse.ArgumentParser):
     def _check_value(self, action, value):
         if isinstance(value, list):
             # converted value must be subset of the choices (if specified)
-            if action.choices is not None and not set(value).issubset(action.choices):
+            empty = value == ['']
```

Comment:
Are you sure about this? You are setting `const` to `[]`. Also, shouldn't you compare to `const`?

Reply:
Good point, `const` should also be set to `['']` (just tested and with an empty list, it would work properly.

Reply:
You mean it doesn't work properly with empty list? Why?

Reply:
`rebase-helper --pkgcomparetool=` is interpreted the same way as `rebase-helper --pkgcomparetool=""`. In such case, `value` is equal to `[""]` so comparing to empty list wouldn't work

Reply:
Ok.